scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -44,7 +47,8 @@
             [mesomatic.scheduler :as msched]
             [mesomatic.types :as mtypes]
             [metrics.timers :as timers]
-            [plumbing.core :as pc])
+            [plumbing.core :as pc]
+            [cook.mesos.task :as task])
```

Comment:
This still used? And lexiconographic.

Reply:
yes `task` is used but `kcc` and `mcc` are no longer used. edited for lexicographic as well